cmd/okctl/deletetestcluster.go

```diff
@@ -165,13 +165,20 @@ $ kubectl get service --all-namespaces
 				return formatErr(err)
 			}
 
+			// TODO --- yeah, this is hardcoded, can we get it from store?
 			err = services.Cluster.DeleteCluster(o.Ctx, api.ClusterDeleteOpts{
 				ID: id,
+				FargateProfileName: "fp-default",
 			})
 			if err != nil {
 				return formatErr(err)
 			}
 
+			err = services.ALBIngressController.DeleteAdditionalALBs(o.CloudProvider, o.Ctx, id)
```

Comment:
Deleting additional ALBs doesn't strictly have anything to do with the ALB ingress controller.

Reply:
This is true, it will be moved :+1: